src/test/java/com/flightstats/hub/dao/aws/S3WriteQueueTest.java

```diff
@@ -0,0 +1,110 @@
+package com.flightstats.hub.dao.aws;
+
+import com.flightstats.hub.app.HubProperties;
+import com.flightstats.hub.metrics.StatsdReporter;
+import com.flightstats.hub.model.ChannelContentKey;
+import com.flightstats.hub.model.Content;
+import com.flightstats.hub.model.ContentKey;
+import com.flightstats.hub.spoke.SpokeWriteContentDao;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@Slf4j
+public class S3WriteQueueTest {
+
+    private SpokeWriteContentDao spokeWriteContentDao = mock(SpokeWriteContentDao.class);
```

Comment:
Damon showed me a `@Mock` annotation to use here instead of explicitly calling `mock()` which I thought was pretty cool. :shrug:

Reply:
It simplifies things a bit, I think.  ClusterContentServiceTest has this, if you're interested.

Reply:
👍